docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -17,19 +17,7 @@ You can use the MSAL.js library with nested app authentication to use single sig
 
 ## NAA supported accounts and hosts
 
-NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support [Azure Active Directory B2C](/azure/active-directory-b2c/overview) for business-to-consumer identity management scenarios. The following table explains the current support by platform. Platforms listed as generally available (GA) are ready for production usage in your add-in.
-
-| Application | Web        | Windows                                              | Mac        | iOS/iPad           | Android        |
-|-------------|------------|------------------------------------------------------|------------|--------------------|----------------|
-| Excel       | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
-| Outlook     | GA         | GA | GA         | GA (iOS)           | GA             |
-| PowerPoint  | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
-| Word        | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
-
-> [!IMPORTANT]
-> To use NAA on platforms that are still in preview (Word, Excel, and PowerPoint), join the [Microsoft 365 Insider Program](https://aka.ms/MSFT365InsiderProgram) and choose **Current Channel (Preview)**. Don't use NAA in production add-ins for any preview platforms. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
-
-For information on using NAA in Microsoft Teams, see [Nested app authentication in Microsoft Teams](/microsoftteams/platform/concepts/authentication/nested-authentication).
+NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identities. It doesn't support [Azure Active Directory B2C](/azure/active-directory-b2c/overview) for business-to-consumer identity management scenarios. For more information on NAA requirements, see [Nested app auth requirement set](/javascript/api/requirement-sets/common/nested-app-auth-requirement-sets).
 
 ## Register your single-page application
 
```

Comment:
@davidchesnut  Do we need to add a note/line about needing another redirect to your actual domain (like a normal SPA app) for when on the web in WXP, given brk-multihub only covers the host applications and OWA which do brokering?

Reply:
I added a note about this later as well as steps in the app registration process.